css/styles.css

```diff
@@ -0,0 +1,710 @@
+  align-items: center;
+}
+
+.main_nav_container {
+  min-height: 4.3751rem;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 1.25rem;
+}
+
+.nav_bar_container {
+  position: fixed;
+  left: -100%;
+  top: 0;
+  gap: 0;
+  display: flex;
+  flex-direction: column;
+  padding: 1.5625rem;
+  background-color: rgba(0, 0, 0, 0.786);
+  backdrop-filter: blur(8px);
+  width: 100%;
+  height: 100%;
+  text-align: left;
+  transition: 0.3s;
+}
+
+.main_nav_link {
+  display: flex;
+  flex-direction: column;
+  margin-top: 3.75rem;
+  font-family: var(--ff-lato);
+}
+
+.main_nav_link li {
+  width: 95%;
+  padding: 1.5625rem;
+  border-top: 1px solid rgba(228, 228, 228, 0.295);
+  transition: 0.5 ease;
+}
+
+.main_nav_link:nth-last-child() {
+  border-bottom: 1px solid rgba(228, 228, 228, 0.295);
+}
+
+.sub_nav_link {
+  transition: 0.7s ease;
+  color: #fff;
+  margin: 1rem 0;
+  font-size: 1em;
+}
+
+.sub_nav_link:hover {
+  color: var(--lineColor);
+}
+
+.sub_about_link {
+  transition: 0.7s ease;
+  color: #fff;
+  margin: 1rem 0;
+  font-size: 1em;
+}
+
+.sub_about_link:hover {
+  color: var(--lineColor);
+}
+
+.sub_home_link {
+  transition: 0.7s ease;
+  color: #fff;
+  margin: 1rem 0;
+  font-size: 1em;
+}
+
+.sub_home_link:hover {
+  color: var(--lineColor);
+}
+
+.menu-sections li:hover {
+  border-top: 1px solid rgba(228, 228, 228, 0.605);
+  border-bottom: 1px solid rgba(228, 228, 228, 0.605);
+}
+
+.hamburger_menu {
+  display: block;
+  cursor: pointer;
+}
+
+.ham_bar {
+  display: block;
+  width: 1.5625rem;
+  height: 0.1875rem;
+  margin: 0.3125rem auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: black;
+}
+
+.hamburger_menu.active .ham_bar:nth-child(1) {
+  transform: translateY(8px) rotate(45deg);
+  background-color: #fff;
+}
+
+.hamburger_menu.active .ham_bar:nth-child(2) {
+  opacity: 0;
+}
+
+.hamburger_menu.active .ham_bar:nth-child(3) {
+  transform: translateY(-8px) rotate(-45deg);
+  background-color: #fff;
+}
+
+.nav_bar_container.active {
+  left: 0;
+}
+
+.ham_bar-1 {
+  display: block;
+  width: 1.5625rem;
+  height: 0.1875rem;
+  margin: 0.3125rem auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: #fff;
+}
+
+.top_section_container {
+  display: flex;
+  flex-direction: column;
+  gap: 0.4375rem;
+  text-align: left;
+  padding: 1.5625rem 1.5625rem 0 1.5625rem;
+}
+
+.main_section_title {
+  font-family: var(--ff-lato);
+  font-size: clamp(0.9rem, 1rem + 5vw, 2.5625rem);
+  color: var(--lineColor);
+}
+
+h1 {
+  font-family: var(--ff-goose);
+  line-height: 3.4375rem;
+  font-size: clamp(0.5rem, 1rem + 10vw, 2.125rem);
+  font-weight: 600;
+  color: #ff422d;
+  background-color: var(--mainColor);
+  background-size: cover;
+  background-blend-mode: multiply;
+  background-clip: text;
+  background-position-y: bottom;
+  margin-bottom: 0.4375rem;
+}
```

Comment:
- [ ] As suggested by the reviewer [here](https://github.com/3nityGitHub/CAPSTONE_PROJECT_1/pull/1#discussion_r1179528861) Please make sure to follow the design guidelines and fix the heading 🛠 You can go through [this link](https://plantpot.works/2886) to get an idea of how you can do that. See the images below for more clarity 💡

Your design previously | Your design now 
---| ---|
![image](https://user-images.githubusercontent.com/106119105/235028216-37c6fecd-b3b7-4a8a-b07f-7d4c0f974375.png) | ![image](https://user-images.githubusercontent.com/106119105/235028377-c944500d-5115-4924-a77b-a6ff7e25663b.png)


- [OPTIONAL] What do you think of using the same background image you was using before? 💭 I think that was looking beautiful 🤩 Since you already had one, please consider reverting back this change, and also because the previous reviewer didn't asked to change it so you can continue with that ✨